### PR TITLE
Implement version via CLI in MA

### PR DESCRIPTION
**Pull request by pskiran1**

### CODE REVIEW

**Review by tgerdesnv — COMMENTED** [review]
Can you upload a picture of the command being run and the output?

File: model_analyzer/cli/cli.py [hunk 1]
```diff
@@ -124,6 +132,23 @@ def _add_config_arguments(self, subparser, config):
                     **config_field.parser_args(),
                 )
 
+    def _show_ma_version(self):
+        """
+        Displays the current version of Model Analyzer and exits.
+        """
+        if len(sys.argv) > 2:
+            print(
+                'Error: Invalid combination of arguments: --version cannot be used with other arguments')
```
[the remark this answers is not in the document]
Comment (pskiran1) [reply model_analyzer/cli/cli.py]: Done 👍

File: model_analyzer/cli/cli.py [hunk 2]
```diff
@@ -124,6 +132,18 @@ def _add_config_arguments(self, subparser, config):
                     **config_field.parser_args(),
                 )
 
+    def _show_ma_version(self):
```
[the remark this answers is not in the document]
Comment (pskiran1) [reply model_analyzer/cli/cli.py]: Updated function name 👍

**Comment by pskiran1** [comment]
> 

Please find the sample here

image

**Pull request merged**